src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -666,6 +685,8 @@ tests:
                 value: 0x0001
               - name: "SceneID"
                 value: 0x01
+      response:
+          error: 0x8b
```

Comment:
Why is this not `error: NOT_FOUND` like I suggested?  I mean, both work, but the one that's not a random hex number is a bit clearer...